src/features/content-insights/charts/bar/Bar.tsx

```diff
@@ -11,12 +11,20 @@ const DEFAULT_SIZE = 50;
 interface Props {
     color?: string;
     direction?: Direction;
-    onMouseEnter?: () => void;
+    label?: string | boolean;
```

Comment:
this is my fault, this should really only be a `string`. could we do the following?

1. let's remove the boolean, it's odd that it could be both a string and boolean
```suggestion
    label?: string;
```
.
2. change this from `label = false,` to just `label,` and let it be undefined
https://github.com/box/box-ui-elements/blob/d8315b512e89b7bc4539d93a6f5c988712b789ba/src/features/content-insights/charts/bar/Bar.tsx#L25
.
3. change this to `hasAxisLabel ? datum[labelAccessor] : ''`
https://github.com/box/box-ui-elements/blob/d8315b512e89b7bc4539d93a6f5c988712b789ba/src/features/content-insights/charts/bar/BarChart.tsx#L75

sorry bout that!